src/workerd/api/node/util.h

```diff
@@ -212,11 +212,11 @@ class UtilModule final: public jsg::Object {
     kj::String functionName;
     kj::String scriptName;
     int lineNumber;
-    int column;
+    int columnNumber;
 
     JSG_STRUCT(functionName, scriptName, lineNumber, column);
   };
-  kj::Array<CallSiteEntry> getCallSite(jsg::Lock& js, int frames);
+  kj::Array<CallSiteEntry> getCallSites(jsg::Lock& js, int frames);
```

Comment:
If this already went out in a release we'll need to continue to support both `getCallSite()` and `getCallSites()` unfortunately. You can introduce a compatibility flag to expose one or the other or just export both in parallel as aliases.